### DISCUSSION TITLE
Theme Colors for custom fluent_ui controls

### DISCUSSION THREAD

**Discussion by ronnyek** [bdlukaa/fluent_ui#1034] [Q&A] [answered]
Greetings,

I've built some custom controls, and want to see if there are colors provided by current theme I can use as defaults for my components.

![image](https://github.com/bdlukaa/fluent_ui/assets/584713/b9db8ac6-53ab-4da1-94e8-aae4010c9a39)
![image](https://github.com/bdlukaa/fluent_ui/assets/584713/78a87576-1b8b-437c-a6c2-825f9b8ec936)

I assume there is a text style I can get from current theme that will automatically make text rendered, update when color scheme changes? Also things like borders vs typical control backgrounds etc.

If I can get some pointers, I can create a pr to document these in colors section of the demo app or something.

Appreciate input, and loving working with fluent_ui

**Comment by ronnyek** [comment]
I've already realized the text style I can pull from FluentTheme.of(context).typography
![image](https://github.com/bdlukaa/fluent_ui/assets/584713/f53a0705-fff7-4ebd-978a-e2d365549f59)

I dont know what to use for border colros to make the spaces between cells more evident.

For the days I was hoping to use a sort of disabled state for the days of other month, and current day will be accent date or selected color if that is different

**Reply by bdlukaa** [reply, marked as answer]
You may want to play with color resources: `FluentTheme.of(context).resources...`. `...resources.controlStrokeColorDefault`, for example, is the default color for controls borders, but there are several options. These colors are used on the built-in widgets, but anyone can make use of them.
